client/components/domains/connect-domain-step/connect-domain-step-suggested-start.jsx

```diff
@@ -9,35 +9,43 @@ import {
 	stepSlug,
 } from 'calypso/components/domains/connect-domain-step/constants';
 import MaterialIcon from 'calypso/components/material-icon';
+import { isSubdomain } from 'calypso/lib/domains';
 import ConnectDomainStepWrapper from './connect-domain-step-wrapper';
 
 import './style.scss';
 
 export default function ConnectDomainStepSuggestedStart( {
 	className,
+	domain,
 	pageSlug,
 	mode,
 	onNextStep,
 	progressStepList,
 	setPage,
 } ) {
 	const { __ } = useI18n();
-	const switchToAdvancedSetup = () => setPage( stepSlug.ADVANCED_START );
+	const firstStep = isSubdomain( domain )
+		? stepSlug.SUBDOMAIN_ADVANCED_START
+		: stepSlug.ADVANCED_START;
+	const switchToAdvancedSetup = () => setPage( firstStep );
+
+	const message = isSubdomain( domain )
+		? __(
+				'This is the easiest way to connect your subdomain, using NS records. If needed you can also use our <a>advanced setup</a>, using CNAME records.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'This is the easiest way to connect your domain, using name servers. If needed you can also use our <a>advanced setup</a>, using root A & CNAME records.' )` *ES Score: 7*